diff --git a/aslam_offline_calibration/kalibr/python/kalibr_calibrate_cameras b/aslam_offline_calibration/kalibr/python/kalibr_calibrate_cameras
index aa957e2..2081a26 100755
--- a/aslam_offline_calibration/kalibr/python/kalibr_calibrate_cameras
+++ b/aslam_offline_calibration/kalibr/python/kalibr_calibrate_cameras
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+print "Using PDF as matplotlib backend"
+import matplotlib as mpl
+mpl.use("PDF")
 print "importing libraries"
 import rosbag
 import cv_bridge
diff --git a/aslam_offline_calibration/kalibr/python/kalibr_calibrate_imu_camera b/aslam_offline_calibration/kalibr/python/kalibr_calibrate_imu_camera
index 69ae426..cbd1fe6 100755
--- a/aslam_offline_calibration/kalibr/python/kalibr_calibrate_imu_camera
+++ b/aslam_offline_calibration/kalibr/python/kalibr_calibrate_imu_camera
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+print "Using PDF as matplotlib backend"
+import matplotlib as mpl
+mpl.use("PDF")
 print "importing libraries"
 import sm
 import kalibr_common as kc
